packages/next-safe-action/src/hooks.ts

```diff
@@ -118,7 +118,7 @@ const useActionCallbacks = <const S extends Schema, const Data>(
  * {@link https://next-safe-action.dev/docs/usage-from-client/hooks/useaction See an example}
  */
 export const useAction = <const S extends Schema, const Data>(
-	safeAction: SafeAction<S, Data>,
+	safeAction: SafeAction<[S], Data>,
```

Comment:
For now, I kept hook functionality as is, meaning that you can only use the hooks with actions that have exactly one input.